packages/fx-core/src/plugins/resource/localdebug/certificate.ts

```diff
@@ -14,284 +22,307 @@ import * as ps from "./util/process";
 const continueText: string = "Continue";
 const learnMoreText: string = "Learn More";
 const learnMoreUrl: string = "https://aka.ms/teamsfx-ca-certificate";
-const confirmMessage: string = "To debug applications in Teams, your localhost server must be on HTTPS.\
+const confirmMessage: string =
+  'To debug applications in Teams, your localhost server must be on HTTPS.\
  For Teams to trust the self-signed SSL certificate used by the toolkit, a CA certificate must be added to your certificate store.\
- You may skip this step, but you'll have to manually trust the secure connection in a new browser window when debugging your apps in Teams.\
- For more information \"https://aka.ms/teamsfx-ca-certificate\". You may be asked for your account credentials when installing the certificate.";
+ You may skip this step, but you\'ll have to manually trust the secure connection in a new browser window when debugging your apps in Teams.\
```

Comment:
Why this "\" added?

Reply:
And I remember we should use `"` for string, not `'`

Reply:
It's auto generated by prettier...

Reply:
So what's the expected result, `"` or `'`?

Reply:
I think we are using ".

Reply:
Agree... then how to make sure next time it won't be auto generated to `'`...